smart_card_connector_app/src/_locales/en/messages.json

```diff
@@ -35,6 +35,10 @@
     "message": "ADD READER",
     "description": "Title of the button that can be used to open a dialog for adding a new Smart Card reader"
   },
+  "addAnotherDevice": {
+    "message": "ADD ANOTHER READER",
+    "description": "Title of the button that can be used to open a dialog for adding another Smart Card reader"
```

Comment:
nit: As above, I would make it explicit that this applies only when there is at least one reader already.